### ISSUE TITLE
Bottom sheet position in landscape mode not stable

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12655]
### Describe your problem!

I lately used c:geo on my tablet for cache planning.
While using the live map I was wondering about two things:
- The bottom sheets (for quick settings as well as for routes) are shown left bound. Technically its no problem, so just asking whether this is intended (and/or as per material design)?
- Sometimes however the bottom sheet for routes suddenly shows centered. Not sure under what conditions (seems only to happen if the last route has been removed).

### How to reproduce?

- Open bottom sheet for routes -> It shows up left bound
- Remove routes or do some other interaction. Under some condition is suddenly jumps to center.

### Actual result after these steps?

![Screenshot_20220130-202802_cgeo](https://user-images.githubusercontent.com/949669/151782713-bcbb5f78-79c9-4568-bd99-e20eec439779.jpg)
![Screenshot_20220130-202735_cgeo](https://user-images.githubusercontent.com/949669/151782718-f33d5481-2067-4ed6-be55-4aa276b97964.jpg)

### Expected result after these steps?

Consistent placing either left or centered. Centered would be my personal visual preference.

### Reproducible

Unclear

### c:geo Version

2022.01.31-NB

### System information

```text
Samsung Galaxy Tab S4, Android 10
```

### Additional Information

The bottom sheet for quick settings is always shown left bound (AFAICS). If we decide to center bottom sheets, this should work for both bottom sheets.

**Comment by moving-bits** [comment]
Very strange. I can reproduce it sometimes on an emulated tablet, but not on a mobile. And only in landscape mode, as in portrait mode the dialog seems to be centered horizontally anyway.

In our code I've found nothing that even tries to influence the horizontal positioning. And I've found no docs stating how to do this.

Theoretically it should be as simple as `dialog.getWindow.setGravity(Gravity.CENTER_HORIZONTAL)` in `Dialogs.bottomSheetDialogWithActionbar()`, but neither this nor several variations in placing, referenced object or derived view seems to work, the most I could get was some sort of flickering.
